content/en/logs/log_collection/java.md

```diff
@@ -65,18 +65,21 @@ Add a new file appender to `log4j.xml`:
 </appender>
 ```
 
-**Inject trace IDs in your logs**
+## Connect your service across Logs and APM
```

Comment:
I believe we should keep the word `traces`.

We are not saying Log Management here as we are talking about the data points, not the products. So we are connecting the traces and the logs, not Log Management and APM.